process/graduation_criteria.adoc

```diff
@@ -23,7 +23,9 @@ To graduate from sandbox or incubating status, or for a new project to join as a
 
  * Have committers from at least two organizations.
  * Have achieved and maintained a Core Infrastructure Initiative https://bestpractices.coreinfrastructure.org/[Best Practices Badge].
+ * Have completed an independent and third party security audit with results published (i.e., https://github.com/envoyproxy/envoy#security-audit)
```

Comment:
nit: e.g, not i.e.
We will need to write down in a bit more detail (elsewhere) what the requirements of the audit are.  To start with, something like "of similar scope and quality as the following examples" might suffice.

Reply:
Also, lets be clear whether the critical vulnerabilities identified in such an audit need to be addressed before graduation.

Reply:
And whether or not such an audit needs to be performed periodically, e.g. annually.

Reply:
thanks, tried to address your concerns with an updated sentence

Reply:
We seem to still be missing wording around whether critical vulnerabilities need to be addressed before graduation, and whether periodic (or per release) audits are required to remain in "graduated" state.  Given that the wording is missing, I assume that the implied answer to both of the above is "no", or "at the discretion of the TOC".  I would strongly encourage making that explicit, to avoid confusion.

Reply:
I agree with @quinton-hoole-2 that we should make these explicit. IMO requiring that critical vulns be fixed prior to graduation is a given. Personally, I think that we should at least "highly encourage" periodic security audits, although I recognize funding needs to be considered.